packages/polling-controller/src/PollingController.ts

```diff
@@ -183,5 +183,8 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
   return PollingControllerBase;
 }
 
+class Empty {}
+
+export const PollingControllerBase = PollingControllerMixin(Empty);
```

Comment:
This is a bit confusing, I think, because it implies either a) that `PollingControllerBase` contains things that `PollingController` and `PollingControllerV2` don't, i.e. these classes somehow extend from `PollingControllerBase`, or b) that `PollingControllerBase` extends from `BaseController.

Currently the suffix that we've attached to PollingController seems to indicate what other functionality PollingController has in it, i.e., V1 indicates that it includes BaseControllerV2. Perhaps to keep the same strategy, we use `PollingControllerOnly` instead?

``` suggestion
export const PollingControllerOnly = PollingControllerMixin(Empty);
```